lib/datasource/terraform-module/index.ts

```diff
@@ -65,16 +70,23 @@ export async function getReleases({
     'terraform.getDependencies()'
   );
   const cacheNamespace = 'terraform-module';
-  const pkgUrl = `${registry}/v1/modules/${repository}`;
+  const cacheURL = `${registry}/${repository}`;
   const cachedResult = await packageCache.get<ReleaseResult>(
     cacheNamespace,
-    pkgUrl
+    cacheURL
   );
   // istanbul ignore if
   if (cachedResult) {
     return cachedResult;
   }
   try {
+    const serviceDiscovery = (
```

Comment:
To prevent doing this query for every repository, we should cache this for a period like 24 hours.

Reply:
Good point.